operators/admin/olm-restricted-networks.adoc

```diff
@@ -34,14 +34,12 @@ link:https://access.redhat.com/articles/4740011[]
 * xref:../../operators/operator_sdk/osdk-generating-csvs.adoc#olm-enabling-operator-for-restricted-network_osdk-generating-csvs[Enabling your Operator for restricted network environments]
 
 include::modules/olm-understanding-operator-catalog-images.adoc[leveloffset=+1]
-include::modules/olm-building-operator-catalog-image.adoc[leveloffset=+1]
-.Additional resources
-
-* xref:../../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[Creating a mirror registry for installation in a restricted network]
-
 include::modules/olm-restricted-networks-configuring-operatorhub.adoc[leveloffset=+1]
+include::modules/olm-pruning-index-image.adoc[leveloffset=+1]
+include::modules/olm-mirroring-catalog.adoc[leveloffset=+1]
 .Additional resources
 
+* xref:../../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[Creating a mirror registry for installation in a restricted network]
 * xref:../../operators/operator_sdk/osdk-generating-csvs.adoc#olm-arch-os-support_osdk-generating-csvs[Architecture and operating system support for Operators]
 
 include::modules/olm-updating-operator-catalog-image.adoc[leveloffset=+1]
```

Comment:
@adellape it sitll take "oc admin catalog build", so http://file.rdu.redhat.com/~adellape/100620/46_disconnected/operators/admin/olm-restricted-networks.html#olm-updating-operator-catalog-image_olm-restricted-networks should not be correct. 

maybe it should refer to http://file.rdu.redhat.com/~adellape/100620/46_disconnected/operators/admin/olm-managing-custom-catalogs.html#olm-updating-index-image_olm-managing-custom-catalogs

Reply:
Update now to use "Updating an index image" instead:

http://file.rdu.redhat.com/~adellape/100620/46_disconnected/operators/admin/olm-restricted-networks.html#olm-updating-index-image_olm-restricted-networks

Reply:
@adellape Got it, it is OK.